lib/spack/docs/tutorial_buildsystems.rst

```diff
@@ -682,18 +696,11 @@ We see the following:
 
 .. literalinclude:: ../../../lib/spack/spack/build_systems/python.py
    :language: python
-   :lines: 16, 142-345
+   :lines: 19,146-357
    :linenos:
 
 Each of these methods have sensible defaults or they can be overridden.
 
-We can write package files for Python packages using the :code:`Package` class,
```

Comment:
@mamelara is this removed because it is now handled elsewhere? or did you think it distracts?

Reply:
whoops never mind just looked above